### DISCUSSION TITLE
`Get an issue` has no Repository

### DISCUSSION THREAD

**Discussion by zunda-pixel** [community/community#62222] [Apps, API and Webhooks] [answered]
### Select Topic Area

Bug

### Body

[Get an issue API](https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28#get-an-issue)'s Response scheme has `repository`

But actual response has `no repository`

**Comment by byrneh** [comment, marked as answer]
@zunda-pixel the [#get-an-issue](https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28#get-an-issue docs example response and response schema does not contain a field "repository".
You have to provide the input parameter "repo" (the name of the repository without the .git extension.) in the request so it must be known anyway.
The repos_url field returned will also contain the repository name, owner etc.

**Reply by zunda-pixel** [reply]
I expected the API to fetch repository data using just the repository name.

**Reply by byrneh** [reply]
@zunda-pixel you provide the owner and repository name

**Comment by mehulrajdev** [comment]
In the upper-right corner of any page, click your profile photo, then click Settings.

Screenshot of GitHub's account menu showing options for users to view and edit their profile, content, and settings. The menu item "Settings" is outlined in dark orange.

In the "Access" section of the sidebar, click  Billing and plans, then click Plans and usage.

Under "GitHub Packages", view details of your usage for data transfer.

Under "Storage for Actions and Packages", view details of your storage usage for GitHub Actions and GitHub Packages combined. Storage usage data synchronizes every hour.

Optionally, next to "Usage this month", click Get usage report to email a CSV report of storage use for GitHub Actions and GitHub Packages to the account's primary email address.
